examples/postprocess/main.js

```diff
@@ -238,12 +479,20 @@
             this._gui = new window.dat.GUI();
 
             var cb = this._gui.add(this._params, 'composer', this._composerConfigNames).listen();
-            var self = this;
```

Comment:
You removed this line and it broke colorCorrection. add bind(this) in colorCorrection callback or keep a self